### ISSUE TITLE
Add proportional scaling

### ISSUE THREAD

**Issue by diogocapela** [diogocapela/flatdraw#2]
_Typically this is done with the ctrl or shift key while dragging the corner._

_When I placed a circle I immediately pressed control to keep the ratio of the object -- photoshop reflex ahah -- it would be nice to have a similar UX._

**Comment by timsayshey** [comment]
Love this app! Upvoting this feature as it was the only core thing I felt was missing. Should scale proportionally by default then require shortcut to break out of. Also, adding an image should scale to fit the canvas instead of being super huge.

**Comment by diogocapela** [comment]
Exactly, proportionally by default, and while the ctrl or shift key is pressed we break out of it.
